### DISCUSSION TITLE
Response Projections include fragments on all$

### DISCUSSION THREAD

**Discussion by esfomeado** [kobylynskyi/graphql-java-codegen#1035] [General]
Hey, looking at the code it seems that the `all$` generated on a response projection doesn´t include the fragments as well.

Is there any reason for this behaviour?

Thanks

**Comment by kobylynskyi** [comment]
@jxnu-liguobin do you recall if there were any limitations on adding fragments into all$ projection?

**Reply by jxnu-liguobin** [reply]
Sorry, I didn't need fragments at the beginning, so I didn't study fragments in depth

**Comment by esfomeado** [comment]
Then I will go ahead and this functionality in. I will add a flag as well.

**Reply by vidhyamanurwar** [reply]
fragment on all$ available in newer version?

**Reply by esfomeado** [reply]
Yes, but not out of the box.

You need to customize the templates to include the fragments.
